src/components/ReviewCard.jsx

```diff
@@ -1,9 +1,12 @@
+import { Link } from "react-router-dom";
 import "./ReviewStyles.css";
 function ReviewCard({ review }) {
   return (
     <div className="ReviewCard">
       <li key={review.review_id}>
-        <h4>{review.title}</h4>
+        <Link to={`/reviews/${review.review_id}`}>
+          <h4>{review.title}</h4>
```

Comment:
headings tags shouldn't be skipped eg. `<h1>` then `<h3>`, the next tag should be the next one down in the level for accessibility, they're not based on size. Can you double check all your h3 and h4 elements and make them h1/h2/h3s as needed. Don't forget to use WAVE/Lighthouse occasionally to check for issues like this